rstream/client.py

```diff
@@ -131,16 +135,23 @@ def remove_handler(self, frame_cls: Type[FT], name: Optional[str] = None) -> Non
         else:
             self._handlers[frame_cls].clear()
 
+    def get_is_connection_active(self) -> bool:
+        if self._conn is None:
+            return False
+        return True
+
```

Comment:
Actually _is_not_closed was just introduced https://github.com/qweeze/rstream/pull/134 (You can see the very last comments) in order to notify the _run_delivery_handlers tasks that they can complete because a client.close() was invoked. This parameter it doesn't check if the connection was closed for other reasons, so probably better to check for _conn and set it if there is a forced disconnection (for example from the server)

Reply:
It’s reasonable to introduce some flags to represent disconnection or close-operation. 

I just think it weird to use `conn is None` before `read_frame` in listener task, and the conn should be None only if `close` is called.

Reply:
Yes I think we can keep self._is_not_closed for this case as well, I made it as @property and pushed the modification. Put it as resolved